packages/manager/modules/vps/src/veeam/modal/schedule-backup/vps-schedule-backup.html

```diff
@@ -0,0 +1,27 @@
+<oui-modal
+    data-primary-label="{{ ::'vps_configuration_veeam_schedule_backup_action_confirm' | translate }}"
+    data-primary-action="$ctrl.onRescheduledAutomatedBackupConfirmed()"
+    data-primary-disabled="!$ctrl.model.rescheduleTimepicker"
+    data-secondary-action="$ctrl.onRescheduledAutomatedBackupCanceled()"
+    data-secondary-label="{{ ::'vps_configuration_veeam_schedule_backup_action_cancel' | translate }}"
+    data-on-dismiss="$ctrl.onModalCanceled()"
+    data-heading="{{:: 'vps_configuration_veeam_schedule_backup_title' | translate}}"
+    data-loading="$ctrl.isLoading"
+>
+    <p
+        class="oui-modal__text"
+        data-translate="vps_configuration_veeam_schedule_backup_description"
+        data-translate-values="{ serviceName: $ctrl.serviceName }"
```

Comment:
It seems to have no translation values in "vps_configuration_veeam_schedule_backup_description"

Reply:
I got your point.
Thanks.